src/main/java/model/input/file/PlainTextFile.java

```diff
@@ -52,7 +52,8 @@ public DataTable createDataTable() throws IOException {
 					builder.createColumn(entry.getKey(), entry.getValue());
 				}
 			}
-			readLines(scanner);
+			ArrayList<String> lines = readLines(scanner);
+			filterLastRows(lines);
```

Comment:
Wat filterLastRows doet is de regels filteren en ze dan toevoegen aan de builder. Dat kan je beter splitsen in 2 methode, 1 die de regels filters en 1 die ze toevoegt aan de builder.